packages/@uppy/dashboard/types/index.d.ts

```diff
@@ -75,3 +75,16 @@ declare class Dashboard extends UIPlugin<DashboardOptions> {
 }
 
 export default Dashboard
+
+// Events
+
+export type DashboardFileEditStartCallback<TMeta> = (file: UppyFile<TMeta>) => void;
+export type DashboardFileEditCompleteCallback<TMeta> = (file: UppyFile<TMeta>) => void;
+declare module '@uppy/core' {
```

Comment:
Why do we need to declare this module?

Reply:
It's so the various definitions of the `UppyEventMap` interface are additive with the initial `@uppy/core` version, and not treated as if they're from separate modules. Typescript refers to this pattern as [module augmentation](https://www.typescriptlang.org/docs/handbook/declaration-merging.html#module-augmentation).